package.json

```diff
@@ -185,7 +185,8 @@
             "fortran": "cd $dir && gfortran $fileName -o $fileNameWithoutExt && $dir$fileNameWithoutExt",
             "sml": "cd $dir && sml $fileName",
             "mojo": "mojo run",
-            "erlang": "escript"
+            "erlang": "escript",
+            "spwn": "spwn build"
```

Comment:
Seems that it needs to add a `-c`?
![image](https://github.com/formulahendry/vscode-code-runner/assets/1050213/a381fd83-5fd4-4e43-aa6e-5c436abe18a8)

Reply:
![image](https://github.com/formulahendry/vscode-code-runner/assets/83927639/2837f3a1-e9ed-436b-baeb-5a84eb22d908)
It fails without `-c` because it can't find your [Geometry dash](https://steamdb.info/app/322170/) save file

![image](https://github.com/formulahendry/vscode-code-runner/assets/83927639/b1d4000e-7949-4701-8060-4692cb7b329b)